src/java.desktop/share/classes/sun/awt/image/ShortBandedRaster.java

```diff
@@ -802,10 +803,10 @@ private void verify() {
     }
 
     public String toString() {
-        return new String ("ShortBandedRaster: width = "+width+" height = "
-                           + height
-                           +" #numBands " + numBands
-                           +" #dataElements "+numDataElements);
+        return "ShortBandedRaster: width = " + width + " height = "
```

Comment:
Maybe wrap `height` to the next line like it's done in several other files above?
Perhaps, wrap in all -Raster classes for consistency, also it's easier to read when each field is concatenated on its own line.

Reply:
did it